packages/cli/src/commands/init.ts

```diff
@@ -779,11 +792,18 @@ const initRepository = async (directory: string) =>
       if (filesystem.exists(gitDir)) {
         filesystem.remove(gitDir);
       }
-      await system.run('git init', { cwd: directory });
-      await system.run('git add --all', { cwd: directory });
-      await system.run('git commit -m "Initial commit"', {
-        cwd: directory,
-      });
+      if (await isInRepo()) {
+        await system.run('git init', { cwd: directory });
```

Comment:
This looks like it's initialising a repository if it's in a repository, I think the opposite is the desired behaviour?

Reply:
a52e066dc38bea4d199ed10f75127088cd984841